geo/src/lib.rs

```diff
@@ -49,6 +49,7 @@
 //! - **[`Bearing`]**: Calculate the bearing between two points.
 //!
 //! - **[`Destination`]**: Calculate the destination point from an origin point, given a bearing and a distance.
+//! - **[`InterpolateLine`]**: Interpolate a `Point` along a `Line` or `LineString`.
 //! - **[`InterpolatePoint`]**: Interpolate points along a line.
```

Comment:
I've looked at https://docs.rs/geo/latest/geo/algorithm/line_measures/trait.InterpolatePoint.html and this PR a few times, but I'm still fuzzy on the difference -- is the old trait deprecated?

Reply:
I deprecated it in this PR: https://github.com/georust/geo/pull/1321/files#diff-1e6b7ec38d282e44d7225459b81fa17d1ca9b7529763d44d0550c303077f6826R9

But your comment made me realize I should remove the deprecated one from these front page docs!